tests/benchmarks/benchmark_cpu_small.py

```diff
@@ -131,67 +131,53 @@ def build():
         N = 25
         _ = Equilibrium(L=L, M=M, N=N)
 
-    benchmark.pedantic(build, setup=setup, iterations=1, rounds=50)
+    benchmark.pedantic(build, setup=setup, iterations=1, rounds=10)
```

Comment:
reducing the number of rounds will make the statistics more noisy, and may lead to more false positives

Reply:
I agree. For example, in the last benchmark, this PR shows speed improvement for `perturb` but it shouldn't. We can decide on the exact number of rounds, but my intention is to balance the time spent on tests. Previously, these equilibrium initialization tests took more time than `fixed_iter_solve` and `perturb` tests. Given that benchmark workflow started to take around 50mins, I wanted to reduce them from 50, which is a bit overkill.